WooCommerce/src/main/kotlin/com/woocommerce/android/ui/coupons/edit/CouponRestrictionsScreen.kt

```diff
@@ -21,12 +23,20 @@ import java.math.BigDecimal
 @Composable
 fun CouponRestrictionsScreen(viewModel: CouponRestrictionsViewModel) {
     viewModel.viewState.observeAsState().value?.let {
-        CouponRestrictionsScreen(it)
+        CouponRestrictionsScreen(
+            it,
+            viewModel::onMinimumAmountChanged,
+            viewModel::onMaximumAmountChanged
+        )
     }
 }
 
 @Composable
-fun CouponRestrictionsScreen(viewState: CouponRestrictionsViewModel.ViewState) {
+fun CouponRestrictionsScreen(
+    viewState: CouponRestrictionsViewModel.ViewState,
+    onMinimumAmountChanged: (BigDecimal?) -> Unit = {},
```

Comment:
The `BigDecimalTextFieldValueMapper` returns a non-nullable `BigDecimal`, so we can pass a non-value here directly:
```suggestion
    onMinimumAmountChanged: (BigDecimal) -> Unit = {},
```

PS: I did the same error on EditCouponScreen 😅, I'll adjust on a different PR later.